sdk/include/psdk/wincon.h

```diff
@@ -246,12 +246,7 @@ typedef struct _KEY_EVENT_RECORD {
         CHAR  AsciiChar;
     } uChar;
     DWORD dwControlKeyState;
-}
-#ifdef __GNUC__
-/* gcc's alignment is not what win32 expects */
-PACKED
-#endif
-KEY_EVENT_RECORD, *PKEY_EVENT_RECORD;
+} KEY_EVENT_RECORD, *PKEY_EVENT_RECORD;
```

Comment:
Maybe add a static_assert to check that this still does what it should do?

Reply:
:+1: good idea.

Reply:
Implemented.